llama_index/vector_stores/qdrant.py

```diff
@@ -80,13 +104,24 @@ def __init__(
 
         self._collection_initialized = self._collection_exists(collection_name)
 
+        # setup hybrid search if enabled
+        if enable_hybrid:
+            self._sparse_doc_fn = sparse_doc_fn or default_sparse_encoder(
+                "naver/efficient-splade-VI-BT-large-doc"
```

Comment:
I will test some encoders for efficiency vs effectiveness.
for now let us use this one as default

Reply:
Yea, this one seems like a good default, and the notebook will describe how to modify this too